web-api/migration-terraform/main/lambdas/migrations/bug-0039-notice-of-trial-date.js

```diff
@@ -0,0 +1,65 @@
+const createApplicationContext = require('../../../../src/applicationContext');
+const {
+  aggregateCaseItems,
+} = require('../../../../../shared/src/persistence/dynamo/helpers/aggregateCaseItems');
+const {
+  Case,
+} = require('../../../../../shared/src/business/entities/cases/Case');
+const {
+  SYSTEM_GENERATED_DOCUMENT_TYPES,
+} = require('../../../../../shared/src/business/entities/EntityConstants');
+const applicationContext = createApplicationContext({});
+
+const migrateItems = async (items, documentClient) => {
+  const itemsAfter = [];
+
+  for (const item of items) {
+    if (item.pk.startsWith('case|') && item.sk.startsWith('case|')) {
+      const fullCase = await documentClient
+        .query({
```

Comment:
I think there are some cases that are too big for this query alone. You might not get back all of the information you desire. You may have to paginate this query with `.queryFull`

Reply:
@mmarcotte I wonder if there have been issues with past migrations that use this pattern. We've done this a lot in the past (used `query` to fetch all case items).

Reply:
mayyyybe